src/containers/technology/FooterTile.js

```diff
@@ -34,23 +34,15 @@ const Icon = styled.img`
   margin-right: 15px;
 `;
 
-const Body = styled.span`
-  color: ${N2};
-  font-size: 14px;
-  line-height: 19px;
-`;
-
 type Props = {
-  body :string;
+  children :any;
```

Comment:
This should be `children :Node;`. You'll also want to add `import type { Node } from 'react';` to the top.